### PR TITLE
[CosmosDB] Emulator release 2.14.20

### PR DESCRIPTION
**Pull request by niteshvijay1995**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@sajeetharan 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by sajeetharan — APPROVED** [review]
LGTM

**Comment by sajeetharan** [comment]
#sign-off

File: articles/cosmos-db/emulator-release-notes.md [hunk 1]
```diff
@@ -20,18 +20,23 @@ The Azure Cosmos DB emulator is updated at a regular cadence with release notes
...
 
 ## Previous releases
 
 > [!WARNING]
 > Previous versions of the emulator are not supported by the product group.
 
+### ``2.14.19`` (Aug 6, 2024)
+
+- Update Azure Cosmos DB Emulator background services to match the latest online functionality of the Azure Cosmos DB.
+- Update data explorer to support vector search
```
Comment (Court72) [inline articles/cosmos-db/emulator-release-notes.md:38]:
```suggestion
- Update data explorer to support vector search.
```

**Pull request merged**